recipes/shell/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "shell" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/prefix-dev/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz"
+  sha256: "0eb58c974361144279e966b072bc69ab0b885090ca63a60bccab11c29ac44ba9"
+
+build:
+  number: 0
+  script: |
+    set -ex
+    export PATH=$HOME/.cargo/bin:$PATH
+    cd crates/shell
+    cargo install --locked --bins --root {{ PREFIX }} --path .  # [unix]
```

Comment:
You can use this syntax for win and unix (because Jinja works the same!).

It would be good to add `--no-track` to not create overlapping `$PREFIX/crates2.json` files.